src/methods/sep24/createPopup.ts

```diff
@@ -0,0 +1,12 @@
+export const createPopup = (popupUrl: string) => {
+  const url = new URL(popupUrl);
+  const popup = open(url.toString(), "popup", "width=500,height=800");
```

Comment:
As a TODO, we should allow the user to test out the `callback` and `on_change_callback` functionality outlined in SEP-24.

Reply:
Issue to handle this later: https://github.com/stellar/nebula/issues/158